File: Logica-Programacao/Javascript/Faccat/ex024.js
let salario, totalVendas, comissao;

salario = parseFloat(prompt("Salário(R$) fixo: "));
totalVendas = parseFloat(prompt("Valor(R$) total das vendas: "));

if (totalVendas <= 1500) {
    comissao = 0.03;
} else {
    comissao = 0.05;
}

salario = salario + (totalVendas * comissao)

alert("Salário final: R$" + salario);

File: Java/Logica-Programacao/src/faccat/Ex024.java
package faccat;

import java.util.Scanner;

public class Ex024 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Digite o salário fixo do funcionário: R$");
        float salary = sc.nextFloat();
        System.out.print("Valor total das vendas: R$");
        float sellsTotal = sc.nextFloat();
        sc.close();

        float finalSalary;

        if (sellsTotal > 1500) {
            finalSalary = salary + (sellsTotal * 5 / 100);
        } else finalSalary = salary + (sellsTotal * 3 / 100);

        System.out.printf("O salário total do funcionário é de R$%.2f\n", finalSalary);
    }
}